### PR TITLE
implement iOS Framwork "SVGgh" for svg rendering on iOS

### PR DESCRIPTION
**Pull request by Longri**
implement iOS Framwork "SVGgh" for svg rendering on iOS

### CODE REVIEW

**Comment by devemux86** [comment]
`natives` sub-folder is supposed to hold the natives libraries needed at runtime.
It's compressed as separate archive.
So what's the need for all these new stuff, the headers etc. ?

**Comment by Longri** [comment]
that's the iOS framework "SVGgh"!
For compile iOS, this framework will compile into the *.ipa!
All is needed!

**Comment by Longri** [comment]
sorry, I closed on my Fork

**Comment by devemux86** [comment]
Yes but at runtime we'll have the headers inside the app?

**Comment by Longri** [comment]
No! On iOS, we have no jit compiler! iOS has an ahead-of-time compiler

**Comment by devemux86** [comment]
Can you run gradle install and check the `vtm-ios-0.6.0-SNAPSHOT-natives.jar` artifact in your local Maven repository?
That's what will go into Maven Central, will this have the right structure?

**Comment by Longri** [comment]
the created vtm-ios-0.6.0-SNAPSHOT-natives.jar artifact is correct!

But wait with merging! I will write a README.md to explain usage this vtm-ios-0.6.0-SNAPSHOT-natives.jar, and put this to folder vtm-ios. Or is it better to write it to regular README? 

but not today, family is waking up! ;-)

**Comment by devemux86** [comment]
Ok just wanted to double ckeck. :smile: 

We can put child readme notes in a root `docs` subfolder with link in main page.
So you can create a readme with a regular name e.g. `ios.md`.

Also is it about time to remove the "experimental" reference for `vtm-ios`?

File: vtm-ios/src/org/oscim/ios/backend/IosSvgBitmap.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        super(getUIImage(inputStream));
+    }
+
+    //get UIImage from SVG file
+    private static UIImage getUIImage(InputStream inputStream) {
+        String svg = getStringFromInputStream(inputStream);
+        SVGRenderer renderer = new SVGRenderer(svg);
+        CGRect viewRect = renderer.getViewRect();
+
+        float scaleFactor = CanvasAdapter.dpi / 240;
```
Comment (devemux86) [inline vtm-ios/src/org/oscim/ios/backend/IosSvgBitmap.java]: Just noting that in [AndroidSvgBitmap](https://github.com/mapsforge/vtm/blob/master/vtm-android/src/org/oscim/android/canvas/AndroidSvgBitmap.java#L38) I found the `scaleFactor` calculation with 240 to produce small symbols, that's why I used the regular 160dpi as base.

Comment (Longri) [inline vtm-ios/src/org/oscim/ios/backend/IosSvgBitmap.java]: I have copy/paste from Desktop! 
I will check the display with 160dpi and maybe change! On simulator 240dpi looks good.

**Comment by devemux86** [comment]
@Longri do you have additional work on this?

**Comment by Longri** [comment]
No, but I would do this on this week

**Comment by Longri** [comment]
or today

**Comment by devemux86** [comment]
Ok!
Have you review the 160 or 240 above, which is appropriate for iOS?

**Comment by Longri** [comment]
yes! for iPhone 4s the display is ok! But I will check this again when I have a Android devices with similar Display size

**Comment by devemux86** [comment]
@Longri thanks for the work on this!

Squashed and merged via 4c5de7453d986d9ffccb5a5efb40921e2ff607cc.
I also added a link for the guide on the main page.

**Pull request closed without merging**